File: apps/institution-api/Models/InstitutionPhone.cs
using System;
using System.Collections.Generic;

#nullable disable

namespace ProviderApi.Models
{
    public partial class InstitutionPhone
    {
        public int InstitutionPhoneId { get; set; }
        public int InstitutionId { get; set; }
        public Guid? PhoneGuid { get; set; }
        public string PhoneType { get; set; }
        public string PhoneUsage { get; set; }
        public int? ListSequence { get; set; }
        public string CountryCallingCode { get; set; }
        public string PhoneNumber { get; set; }
        public string Extension { get; set; }
        public DateTime? PhoneBusinessStartDate { get; set; }
        public DateTime? PhoneBusinessEndDate { get; set; }

        public virtual Institution Institution { get; set; }
    }
}

File: libs/common/models/institution-phone.ts


import { Institution } from './institution';

export class InstitutionPhone {
    public institutionPhoneId: number = 0;
    public institutionId: number = 0;
    public phoneGuid: string;
    public phoneType: string = '';
    public phoneUsage: string = '';
    public listSequence: number;
    public countryCallingCode: string = '';
    public phoneNumber: string = '';
    public extension: string = '';
    public phoneBusinessStartDate: Date;
    public phoneBusinessEndDate: Date;
    public institution: Institution = new Institution();

    constructor(init? : Partial<InstitutionPhone>) {
        
        Object.assign(this, init);
    }
}

